docs/SMCWG-charter.md

```diff
@@ -196,13 +202,13 @@ forth in the Bylaws.
    Teleconference, or upon the request of any Member of the SMCWG, by a Ballot
    among Members of the SMCWG.
 
-   In order to determine the composition of the initial set of SMCWG Members,
-   at least twenty-four (24) hours prior to the initial meeting of the SMCWG,
-   the SMCWG Chair SHALL publish a list of Members seeking to participate who
-   they determine meet the criteria set forth above. As the first order of
-   business at the first meeting of the SMCWG, those organizations on the
-   Chair’s list of proposed, qualifying Members SHALL vote to determine the
-   initial set of SMCWG Members.
+To confirm the list of initial Members, at least two-thirds of the organizations
+on the Chair's list of proposed, qualifying Members SHALL publicly vote in favor
+of adopting the list of Members. If the initial list fails to be adopted, the
+Chair SHALL consider the feedback and MAY update the initial list of proposed,
+qualifying Members as a result, and a second vote using these same rules shall
+be performed. If an initial list of Members cannot be agreed upon, the SMCWG
```

Comment:
I'm potentially supportive of this improvement, but I'm not sure the procedural time-bomb at the end is useful, especially in terms of number of attempts.  If this is the route we want to go, I'd prefer something along the lines of a time limit to resolve initial membership, like 60 days.  It would allow more flexibility to get it right, in the event there are unforeseen issues.

Reply:
What do you mean by procedural time-bomb? There's no timing requirement between the first and second list, precisely to address your concern. The goal, however, was to limit the number of attempts, to make sure that if it fails the first time (which, I hope we can all agree, is extremely unlikely), whatever caused it to fail will be addressed in a way satisfactory to the 2/3.

The primary goal of this isn't about setting time-bombs though, it's about make sure a quorum of 3 isn't going to try and adopt Draft Guidelines in haste and without considering feedback. Which is a real concern of ours.